### DISCUSSION TITLE
Cannot add environment variables

### DISCUSSION THREAD

**Discussion by Ponjimon** [vercel/vercel#4256] [General] [answered]
![image](https://user-images.githubusercontent.com/770632/81009257-95cf2880-8e54-11ea-835b-8e0dba27040f.png)
![image](https://user-images.githubusercontent.com/770632/81009265-9b2c7300-8e54-11ea-9234-a513a77e86c7.png)
It doesn't send anything lol

**Comment by brunohkbx** [comment]
Up

**Comment by coetry** [comment, marked as answer]
Hi folks, we pushed a fix for this just a while ago. Sorry that you had to run into this!
